drake/util/timeUtil.h

```diff
@@ -0,0 +1,40 @@
+#ifndef DRAKE_WINTIME_H
+#define DRAKE_WINTIME_H
```

Comment:
chrono, not crono. Travis is using GCC 4.6.3 and clang 3.4 I see. I think those have chrono already, but not 100% sure.